src/coprocessor/dag/executor/index_scan.rs

```diff
@@ -49,7 +49,10 @@ impl<'a> IndexScanExecutor<'a> {
             key_ranges.reverse();
         }
         let cols = meta.mut_columns();
-        if cols.last().map_or(false, |c| c.get_pk_handle()) {
+        let has_handle_col = cols.last().map_or(false, |c| {
+            c.get_pk_handle() || c.get_column_id() == EXTRA_HANDLE_COLUMN_ID
```

Comment:
Should we set `pk_handle` true for `EXTRA_HANDLE_COLUMN_ID` ? If not, I think you should process `EXTRA_HANDLE_COLUMN_ID` in  any place `pk_handle` related